### ISSUE TITLE
Running tutorial on different Omniverse kitchenset data- does not fit objects

### ISSUE THREAD

**Issue by jaymefosa** [NVIDIAGameWorks/kaolin#396]
Swapping out the clock files for a kitchenware set of images (10 images) generated by the Kaolin datagenerator in Omniverse, following the Kaolin video tutorial, results in mesh generation that is roughly stuck in the shape of a sphere.  Examples for the 'ladle' mesh completely failed.

Shown here is the pot mask where the handle can't be learned:

![image](https://user-images.githubusercontent.com/92468/119769313-34617800-bef5-11eb-9403-183d8e7f0926.png)

Should the tutorial code be runnable on other data generated using the kitchenset files? I had thought it could overfit to a single mesh example but it's having issues with shapes. My understanding was that the sphere template wasn't a barrier for these kinds of deformations?

**Comment by Caenorst** [comment]
The tutorial should work with other shapes, although I will admit that haven't tested others, there could be a little bit of hyperparameter tuning to do, will be looking at this one
